releasenotes/notes/improve-systemd-config-a82bca0014f6ccc6.yaml

```diff
@@ -0,0 +1,20 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+upgrade:
+  - |
+    Update to the configuration of the systemd check: Before, if `unit_names` and `unit_regexes`
```

Comment:
note: in reStructuredText format, inline code needs to be quoted with double backticks, see http://docutils.sourceforge.net/docs/user/rst/quickref.html#inline-markup